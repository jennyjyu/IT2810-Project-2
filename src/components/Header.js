import React from 'react';

export default function Header(props) {
    return (
        <div className="header-container">
            <header className="header">
            <h1>
                {props.headerText}
            </h1>
            </header>
        </div>
    )
}
