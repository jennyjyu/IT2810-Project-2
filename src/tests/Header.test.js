import React from "react"
import Header from "../components/Header"
import renderer from "react-test-renderer"
// import App from "../App"



test("Header snapshot test", () => {
  const component = renderer.create(
    <Header/>
  )
  let tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
