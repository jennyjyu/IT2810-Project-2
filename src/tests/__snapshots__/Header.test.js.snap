// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot test 1`] = `
<div
  className="header-container"
>
  <header
    className="header"
  >
    <h6 />
  </header>
</div>
`;
